fortinet_fortigate/icon_fortinet_fortigate/connection/connection.py

```diff
@@ -20,6 +20,10 @@ def connect(self, params):
 
         self.session = Session()
         self.session.params = self.session_params
+        if self.host.startswith("http://"):
+            self.host = self.host[7:]
+        if self.host.startswith("https://"):
+            self.host = self.host[8:]
```

Comment:
You could just call lstrip('https://') here too.

We should handle suffix too. Users just want to be able to copy and paste the URL from their Fortinet instance. That gives me this: https://fortigate-vm02.vuln.lax.rapid7.com/ng/

rstrip ending / and anything after